bucket/deskpins.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.32",
+    "description": "Make any application always on top.",
+    "homepage": "https://efotinis.neocities.org/deskpins",
+    "license": "MIT",
```

Comment:
![image](https://user-images.githubusercontent.com/13260377/91258252-3d59c900-e76c-11ea-8c2d-a208dd467aac.png)

Reply:
> Notes:
> * license: The MIT license can be found at license.txt of the source code package.

The author puts MIT license in the source code archive. Should I change *license* to "Freeware", or just keep it as "MIT"?

Reply:
How is the source code license related to the released binary, when there is specific license for the final binary?

Reply:
OK. I understood. I will change *license* to "Freeware".